### DISCUSSION TITLE
Custom Form / Email option

### DISCUSSION THREAD

**Discussion by jeromeds-123** [liedekef/events-made-easy#160] [Q&A] [answered]
Dear Francky,

Is there an method to have booking info dynamically fed into a form (I use Contact Foms 7, and am open to change) or have a button event-triggered email sent by EME ?

My issue is as follows : we accept direct bak transfer, aka offline payment. These are randomly used. I would like the payment page to display a "Click here if you choose to pay by bank transfer" button. This button would trigger an email to the event organier, signalling to manually approve the booking. I would like to avoid a mailto link which would open the end-user email client to send the email, if possible.

My attempts have been : 

- Feed EME shortcodes into the Contact Forms email
- Feed EME shortcodes into the Contact Forms form
- Fiddling with Contact Forms 7 Dynamic Text Extension

Any other way to achieve this ? 

If not I could also ask the end user to reply to the RSVP Pending email they receive...

Regards,

Jerome

**Comment by liedekef** [comment]
I configured EME to have a payment button for online payment + the events to have auto-approve after payment. In the payment header template (or the offline payment-configuration option) I also mention how to pay offline. This info is also shared via the RSVP-pending mail (and in fact, there I also point to the payment form if they still decide to pay online).

**Reply by jeromeds-123** [reply]
Dear Francky,
Thenk you for your fast answer.
My issue is not with getting information to the user on how to pay. This is very clear, well implemented and documented.
My issue (or rather my wish) is to have an easy option for our customer to (manually) inform us they have chosen for offline payment (rather than other options). Ideally a button which triggers an email to the event organiser, email containing relevant information (event id, customer name, amount to be paid, seats booked, etc.)
This is what I was exploring with Contact Forms : pre-filling the contact form with EME shortcodes. Which unfortunately does not work.

Or can I give a link to the customer to manually approve the booking ? Kind of like the Confirmation link, but to impact the Approve status ? We would then, on the back end, manually validate the Paid status of the booking.

Thank you for your time,
Kind regards, 
Jerome

**Reply by liedekef** [reply]
That's why I said to use the auto-approve: those that choose to pay online will get auto-approved, those that don't pay online stay in state pending. So you know who did not pay online that way. And you can still chose to approve those bookings based upon internal rules.

**Reply by jeromeds-123** [reply]
We are using the auto-approve option.
My wish is to easily distinguish an abandoned (unpaid) booking, and a offline-paid booking. My thought was to give our customer the possibility to one-click signal the event organiser of his choice. 
Just like the on-line payment triggers the approval & linked mails, I was looking for a way to generate this mail (with EME content inside) through an (easy) customer action.

**Reply by liedekef** [reply, marked as answer]
When someone books, they will never follow-up on an action to tell you they will pay offline (my experience).
If you want people to indicate that they will pay offline, create a custom field (although they are still able to pay online then ...).
For me: once a booking is pending, it is offline payment.
You can configure a reminder mail for bookings that need approval to tell the people that their unpaid booking will get deleted after X time, upon which you can ask for a reply if they want to pay offline and as such keep the booking (and then you can approve it). The "delete after X time" can be set in the "Planned actions" menu (is global), and is in fact in minutes.
